content/cpp/concepts/math-functions/terms/exp/exp.md

```diff
@@ -1,43 +1,52 @@
 ---
 Title: 'exp()'
-Description: 'Returns e raised to the power of the argument.'
+Description: 'Returns the base-e exponential function of the argument.'
 Subjects:
   - 'Computer Science'
 Tags:
   - 'Functions'
   - 'Arithmetic'
+  - 'Methods'
 CatalogContent:
   - 'learn-c-plus-plus'
   - 'paths/computer-science'
 ---
 
-`exp()` returns the base-_e_ exponential function of the argument, which is _e_ raised to the power of the argument.
-
-The number _e_, also known as Euler's number, is a mathematical constant approximately equal to 2.71828 and the base of the natural logarithm.
-
-The `cmath` library must be added to the top of the file with `#include <cmath>`.
+The **exp()** function returns the base-_e_ exponential value of the argument (i.e., _e_ raised to the power of the argument). This is the inverse of the [`log()`](https://www.codecademy.com/resources/docs/cpp/math-functions/log) function.
 
 ## Syntax
 
-Returns *e*ⁿ:
-
 ```cpp
```

Comment:
```suggestion
```pseudo
```

The \```pseudo code snippet is commonly used in the "Syntax" section of an entry.